.github/workflows/continuous-build.yml

```diff
@@ -64,5 +64,20 @@ jobs:
          -Dsonar.python.pylint.reportPaths=pylintreport.txt
          -Dsonar.projectBaseDir=./app
          -Dsonar.coverage.exclusions=tests/**/*
-
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
maybe add a `requires: [build]`?

Reply:
good shout let me have a go at that

Reply:
I used `needs:` - [see docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idneeds)